src/Raven.Server/Documents/Sharding/Operations/IShardedReadOperation.cs

```diff
@@ -27,13 +27,27 @@ public interface IShardedReadOperation<TResult, TCombinedResult> : IShardedOpera
             CombinedEtag = actualEtag
         };
 
+        var span = commands.Span;
+
         if (ExpectedEtag == result.CombinedEtag)
         {
-            result.StatusCode = (int)HttpStatusCode.NotModified;
-            return result;
+            var allNodModified = true;
```

Comment:
`allNodModified` -> `allNoneModified`

Reply:
Fixed in https://github.com/ravendb/ravendb/pull/14945